cloud-init/gophish-dir-setup.tpl.yml

```diff
@@ -0,0 +1,21 @@
+---
+runcmd:
+  # Create a subdirectory on the EFS share for GoPhish data
+  - [mkdir, -p, "${efs_gophish_data_dir}"]
+  # Set group ownership of the gophish data directory to the gophish group
+  - [chown, --verbose, --recursive, ":gophish", "${efs_gophish_data_dir}"]
+  # Give full permissions on the gophish data directory to the gophish group
+  - [chmod, "0775", "${efs_gophish_data_dir}"]
+  # Set the group sticky bit on the gophish data directory so that future
+  # content in the directory will also be owned by the gophish group
+  - [chmod, g+s, "${efs_gophish_data_dir}"]
```

Comment:
This could be simplified to:
`- [chmod, "2775", "${efs_gophish_data_dir}"]`

Reply:
Rectified in https://github.com/cisagov/cool-assessment-terraform/pull/45/commits/adcc21bfe4a7dc081d7e9ea2ae0bf106cef66a4a - thanks!